src/main/java/Pebble/Parser.java

```diff
@@ -0,0 +1,99 @@
+package pebble;
```

Comment:
Good job on this. I did not know packages had to be in lowercase. All my packages started with a capital :{